student.rb

```diff
@@ -0,0 +1,120 @@
+require 'csv'
+require_relative 'calculator'
+class Student
+
+	attr_accessor :name, :roll_number, :bengali, :english, :maths, :computer, :total, :percentage, :percentile
+	def initialize name, roll, bengali,english,maths,computer,t,per
+		@name=name
+		@roll_number = roll
+		@bengali = bengali
+		@english = english
+		@maths = maths
+		@computer = computer
+		@total=t
+		@percentage=per
+		@percentile=0
+	end
+
+	def calculate_total
+		@total=Calculator.new.add(@total,@bengali)
+		@total=Calculator.new.add(@total,@english)
+		@total=Calculator.new.add(@total,@maths)
+		@total=Calculator.new.add(@total,@computer)
+	end
+
+	def calculate_percentage
+		@percentage=Calculator.new.division(@total,400)
+		@percentage=Calculator.new.multiply(@percentage,100)
+	end
+end
+
+def store_file students
+	File.open("StudentGrade.txt","w") do |file|
+		students.each do |student|
+			file.puts "#{student.name},#{student.roll_number},#{student.bengali},#{student.maths},#{student.english},#{student.computer},#{student.total},#{student.percentage}"
+		end
+	end
+end
+
+def read_file
+	students=[]
+	File.open("StudentGrade.txt","r") do |file|
+		file.each_line do |line|
+			data=line.split(',')
+			students<<Student.new(data[0],data[1].to_i,data[2].to_i,data[3].to_i,data[4].to_i,data[5].to_i,data[6].to_i,data[7].to_f)
+		end
+	end
+	students
+end 
+
+def display_info students
+	students.each do |student|
+		puts "Name: #{student.name}\nRoll Number: #{student.roll_number}\nBengali: #{student.bengali}\nMaths: #{student.maths}\nEnglish: #{student.english}\nComputer: #{student.computer}\nTotal: #{student.total},Percentage: #{student.percentage}"
+		puts "-"*30
```

Comment:
`puts "-"*30` -- Why this?

Reply:
just to make it good to look at when displayed on console